### PR TITLE
AggregateRepository method find()

### PR DESCRIPTION
**Pull request by dartvandru**

### CODE REVIEW

File: src/modules/AggregateRepository.js [hunk 1]
```diff
@@ -167,22 +158,33 @@ export default class AggregateRepository {
...
       const type = event.eventType.split('.').pop();
-
       const applyMethod = this.getApplyMethod(entity, type);
 
       applyMethod.call(entity, event);
     });
-
   }
 
+  find(entityId, options) {
```
Comment (cer) [inline src/modules/AggregateRepository.js]: This should be like other methods and find({entityClass, ....})

File: src/modules/AggregateRepository.js [hunk 2]
```diff
@@ -98,17 +97,18 @@ export default class AggregateRepository {
...
-
         }
       });
   }
 
   createEntity({ EntityClass, command, options }) {
-
-    const entity = new EntityClass();
-
+    let entity;
```
Comment (cer) [inline src/modules/AggregateRepository.js]: This code 105-110 is duplicated and should be extracted into a function

Comment (dartvandru) [reply src/modules/AggregateRepository.js]: Resolved with: https://github.com/eventuate-clients/eventuate-client-nodejs/pull/45/commits/726e8d6663662ee029d2e21c1932787feaa962fd

File: src/modules/AggregateRepository.js (hunk 1 above)
Comment (dartvandru) [reply src/modules/AggregateRepository.js]: Resolved with: https://github.com/eventuate-clients/eventuate-client-nodejs/pull/45/commits/726e8d6663662ee029d2e21c1932787feaa962fd

**Pull request merged**